scripts/simtest/cargo-simtest

```diff
@@ -28,12 +28,12 @@ done
 #   echo "Please commit or revert your changes to Cargo.lock before running cargo simtest"
 #   exit 1
 # fi
-cd "$STARTING_DIR"
+cd "$STARTING_DIR" || exit 1
 
 cleanup () {
-  cd "$MANIFEST_DIR"
+  cd "$MANIFEST_DIR" || return
```

Comment:
is `return` sufficient to make the caller site to exit in error?

Reply:
On `return`
> Causes a function to stop executing and return the value specified by n to its caller. If n is omitted, the return status is that of the last command executed in the function body.

Reply:
Yeah sure `cleanup` will return n but if we don't do anything about it at the caller site, and we don't, then it's useless

Reply:
i changed it